core/templates/base-components/base-content.component.ts

```diff
@@ -144,8 +144,23 @@ export class BaseContentComponent {
 
   acknowledgeCookies(): void {
     let currentDateInUnixTimeMsecs = new Date().valueOf();
+    // This cookie should support cross-site context so secure=true and
+    // sameSite='none' is set explicitly. Not setting this can cause
+    // inconsistent behaviour in different browsers in third-party contexts
+    // e.g. In Firefox, the cookie is accepted with the following warning:
+    // "Cookie “OPPIA_COOKIES_ACKNOWLEDGED” will be soon rejected because it
+    // has the “SameSite” attribute set to “None” or an invalid value, without
+    // the “secure” attribute. To know more about the “SameSite“ attribute,
+    // read
+    // https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite"
```

Comment:
Do we need to paste the whole warning here? IMO it is fine to just mention that it shows some warning Firefox, the comment is already quite long.

Reply:
Sure, I've shortened it. PTAL!